profile/README.md

```diff
@@ -1,9 +1,7 @@
-This is the [community maintained](https://github.com/cdist-community)
-fork of [ungleich](https://github.com/ungleich)'s [cdist](https://github.com/ungleich/cdist)
-(after [f061fb1](https://github.com/ungleich/cdist/commit/f061fb168ddacc894cb6e9882ff5c8ba002fadd8)).
+This is [skonfig](http://skonfig.li/) configuration management, a drop in replacement for [cdist](http://cdi.st/).
```

Comment:
[skonfig](https://skonfig.li) is fork of [ungleich](https://github.com/ungleich)'s [cdist](https://github.com/ungleich/cdist) (after [f061fb1](https://github.com/ungleich/cdist/commit/f061fb168ddacc894cb6e9882ff5c8ba002fadd8)).